includes/MirahezeMagicHooks.php

```diff
@@ -124,33 +143,97 @@ public static function onCreateWikiRename( $dbw, $old, $new ) {
 			}
```

Comment:
This will need updating to rename each container as well.

Reply:
Done.